script.js

```diff
@@ -0,0 +1,75 @@
+function getComputerChoice() {
+    let cpuChoice;
+    let choice = Math.floor(Math.random() * 3) + 1;
+    
+    if (choice === 1) {
+        cpuChoice = "ROCK";
+    }
+    else if (choice === 2) {
+        cpuChoice = "PAPER";
+    }
+    else {
+        cpuChoice = "SCISSORS";
+    }
+
+    return cpuChoice;
+}
+
+function getHumanChoice() {
+    let humanChoice = prompt("Rock, Paper, or Scissors?");
+    humanChoice = humanChoice.toUpperCase();
+
+    if (humanChoice === "ROCK" || humanChoice === "PAPER" || humanChoice === "SCISSORS") {
+        return humanChoice;
+    }
+    else {
+        alert("Invalid input. Try again");
```

Comment:
I feel an alert message that'll let the person know they're to make selection from the provided options will be better than just Try again!